packages/main/cypress/specs/Card.cy.tsx

```diff
@@ -1,24 +1,171 @@
+import Button from "../../src/Button.js";
 import Card from "../../src/Card.js";
 import CardHeader from "../../src/CardHeader.js";
 
+function CardSample() {
+	return <Card id="card">
+		<CardHeader
+			id="cardHeader"
+			slot="header"
+			additionalText="4 of 10"
+			titleText="Quick Links"
+			subtitleText="Quick links sub title"
+			interactive={true}>
+		</CardHeader>
+		<div></div>
+	</Card>;
+}
+
+function CardSample2() {
+	return <Card id="card2">
+		<CardHeader
+			id="cardHeader2"
+			slot="header"
+			additionalText="4 of 10"
+			titleText="Quick Links"
+			subtitleText="Quick Links">
+		</CardHeader>
+		<div></div>
+	</Card>;
+}
+
+describe("Card general interaction", () => {
+	it("Tests initial rendering", () => {
+		cy.mount(<CardSample />);
+
+		// assert
+		cy.get("#card").should("exist");
+	});
+
+	it("Tests interactive header results in interactive class on the card", () => {
+		cy.mount(
+			<div>
+				<CardSample />
+				<CardSample2 />
+			</div>
+		);
+
+		// assert
+		cy.get("#cardHeader")
+			.should("have.prop", "interactive", true);
+		cy.get("#card")
+			.shadow()
+			.find(".ui5-card-root")
+			.should("have.class", "ui5-card--interactive");
```

Comment:
This test is valid - you check that if you set `interactive`, the correct class will be applied.